packages/bisect_ppx/bisect_ppx.1.4.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+
+synopsis: "Code coverage for OCaml"
+license: "MPL2"
+homepage: "https://github.com/aantron/bisect_ppx"
+doc: "https://github.com/aantron/bisect_ppx"
+bug-reports: "https://github.com/aantron/bisect_ppx/issues"
+
+dev-repo: "git+https://github.com/aantron/bisect_ppx.git"
+authors: [
+  "Xavier Clerc <bisect@x9c.fr>"
+  "Leonid Rozenberg <leonidr@gmail.com>"
+  "Anton Bachin <antonbachin@yahoo.com>"
+]
+maintainer: [
+  "Anton Bachin <antonbachin@yahoo.com>"
+  "Leonid Rozenberg <leonidr@gmail.com>"
+]
+
+depends: [
+  "base-unix"
+  "dune" {build}
+  "ocaml" {>= "4.02.0"}
+  "ocaml-migrate-parsetree" {>= "1.1.0"}
+  "ppx_tools_versioned"
+
+  "ocamlfind" {dev}
```

Comment:
These two dev dependencies have been introduced in 1.4.0 (and tests removed). Is that expected?

Reply:
Yes, the tests were removed before the opam 2 upgrade IIRC, when testing with opam was recursive.